docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[fleet-api-docs]]
+= {{fleet} API Docs}
+
+{fleet} is fully supported by an API layer.  Anything you can do in the UI, can be
+done using the API.  Indeed, the UI consumes the API and proves out its usage!
```

Comment:
```suggestion
{fleet} is fully supported by an API layer. Any actions you can perform
through the {fleet} UI are also available through the API. In fact, the UI
consumes the API.
```

Reply:
if you don't prefer the language I used, it was copied from another page I think, I can look for it and update it there for us too.  Thanks for the rewrite.

Reply:
@EricDavisX I think the language has a few problems. It's too vague because it doesn't qualify which UI we're talking about. Generally it's good to avoid words like "thing" because they are too broad and don't translate well. Also, the word "anything" tends to over-promise. The last line seems too enthusiastic for docs, but would probably be OK in the blog post.